java/client/src/main/java/glide/api/models/commands/json/JsonGetOptions.java

```diff
@@ -0,0 +1,56 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.json;
+
+import glide.api.commands.servermodules.GlideJson;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+
+/** Additional parameters for {@link GlideJson#get} command. */
+@Getter
+@Builder
+public final class JsonGetOptions {
+    /** ValKey API string to designate INDENT */
+    public static final String INDENT_VALKEY_API = "INDENT";
+
+    /** ValKey API string to designate NEWLINE */
+    public static final String NEWLINE_VALKEY_API = "NEWLINE";
+
+    /** ValKey API string to designate SPACE */
+    public static final String SPACE_VALKEY_API = "SPACE";
+
+    /** Sets an indentation string for nested levels. Defaults to null. */
+    @Builder.Default private String indent = null;
```

Comment:
you don't need to default `null`

Reply:
@Builder.Default requires an initializing expression (' = something;').